frontend/src/v4/routes/viewerGui/components/sequences/sequences.component.tsx

```diff
@@ -60,21 +61,44 @@ interface IProps {
 	draggablePanels: string[];
 	toggleLegend: () => void;
 	resetLegendPanel: () => void;
+	openOnToday: boolean;
 }
 
 const da =  new Date();
 
 const SequenceDetails = ({
 	startDate, endDate, selectedDate, selectedEndingDate, setSelectedDate, stepInterval, stepScale, setStepInterval,
-	setStepScale, currentTasks, loadingFrameState, loadingViewpoint, rightPanels, toggleActivitiesPanel,
+	setStepScale, currentTasks, loadingFrameState, loadingViewpoint, rightPanels, toggleActivitiesPanel, openOnToday,
 	fetchActivityDetails, onPlayStarted, frames, isActivitiesPending, toggleLegend, draggablePanels
-}) => (
+}) => {
+	const [dateToUse, setDateToUse] = useState(null);
+
+	useEffect(() => {
+		if (!openOnToday) {
```

Comment:
- When I have the toggle date on if I change the frame and I close the card when I open it again it gets reset to todays date.
It should keep  selected the last date that I selected:
![bad date](https://github.com/3drepo/3drepo.io/assets/42032433/c7c9b4ca-30e5-4724-bc88-b361278a841d)